### PR TITLE
Fix Uint256 Uint160

### PR DESCRIPTION
**Pull request by Tommo-L**
relate to #1387

### CODE REVIEW

**Comment by vncoelho** [comment]
@Tommo-L, perhaps we could merge this PR on #1387, since @john-h-k and @MithrilMan started the topic.

File: src/neo/UInt256.cs [hunk 1]
```diff
@@ -23,13 +25,10 @@ public UInt256()
...
 
-        public unsafe UInt256(ReadOnlySpan<byte> value)
+        public UInt256(ReadOnlySpan<byte> value)
         {
-            fixed (ulong* p = &value1)
-            {
-                Span<byte> dst = new Span<byte>(p, Length);
-                value[..Length].CopyTo(dst);
-            }
+            Span<byte> dst = MemoryMarshal.AsBytes(MemoryMarshal.CreateSpan(ref value1, Length / sizeof(ulong)));
```
Comment (shargon) [inline src/neo/UInt256.cs]: Why not use the same schema as Uint160?

**Comment by Tommo-L** [comment]
@vncoelho Need your help, I couldn't send commit directly. And #1387 needs some fixes, which didn't pass the ut. Could you add to it, and I'll close this PR.

**Comment by Tommo-L** [comment]
> Why not use the same schema as Uint160?

@shargon Uint160 has two different types(ulong and uint)

**Comment by vncoelho** [comment]
Merged @Tommo-L, I cloned his repo and added yours as upstream.

Then, I checked into his branch `git checkout patch-1` and merged with `git merge upstream fix_uint256_uint160`

**Comment by Tommo-L** [comment]
Thank you 😊

**Pull request closed without merging**